client/state/signup/dependency-store/reducers.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	SIGNUP_DEPENDENCY_STORE_RESET,
+	SIGNUP_DEPENDENCY_STORE_UPDATE_STATE,
+} from 'state/action-types';
+
+export function storeState( state = null, action ) {
```

Comment:
Great suggestion. Updated it to default to an empty object. `null` could have caused some headaches or errors on the console.